pkg/applyconfiguration/testdata/cronjob/api/v1/cronjob_types.go

```diff
@@ -294,8 +294,10 @@ type CronJobSpec struct {
 	unexportedField string
 
 	// This tests that both unexported and exported inline fields are not skipped in the schema generation
-	unexportedStruct `json:",inline"`
-	ExportedStruct   `json:",inline"`
+	//
+	// This currently does not work
+	//unexportedStruct `json:",inline"`
```

Comment:
Are we tracking this limitation and the one below somewhere? (e.g. in an ApplyConfiguration umbrella issue?)